ComSemApp/teacher/views.py

```diff
@@ -355,3 +355,131 @@ def delete_file(url):
         os.remove(url)
     except FileNotFoundError:
         pass
+
+
+class SpeakingPracticeAttemptReviewView(TeacherCourseViewMixin, ListView):
+    """
+    This view displays the SpeakingPracticeReviewRequestListView and the
+    SpeakingPracticeAttemptReviewCreateView/SpeakingPracticeAttemptReviewUpdateView
+    """
+    model = SpeakingPracticeAttempt
+    template_name = "ComSemApp/teacher/attempt_request_review.html"
+
+
+class SpeakingPracticeReviewRequestsListView(TeacherCourseViewMixin, ListView):
+    """
+    This view presents the list of review requests. on tghe SpeakingPracticeAttemptReviewPage 
```

Comment:
typo in docstring `. on thge` should be `on the`. The period should also be removed